src/components/legend/legend-selectors.js

```diff
@@ -68,8 +67,40 @@ const parseLegend = (config) => {
 }
 
 const joinAgricultureTitles = (titles) => {
-  const trimmedTitles = titles.map(title => title.split(" ")[0]);
-  return `${trimmedTitles.join(' and ')} agriculture`; 
+  const nonAgricultureTitles = titles.filter(title => !title.toLowerCase().endsWith('agriculture'))
+  const agricultureTitles = titles.filter(title => title.toLowerCase().endsWith('agriculture'));
+  const trimmedTitles = agricultureTitles.map(title => title.split(" ")[0]);
+  const formattedAgricultureTitles = `${trimmedTitles.join(' and ')} agriculture`
+
+  return `${[...nonAgricultureTitles, formattedAgricultureTitles].join(' and ')}`;
```

Comment:
Since we have four layers, we should tweak this method a bit:
```suggestion
  const nonAgricultureTitles = titles.filter(title => !title.toLowerCase().endsWith('agriculture'))
  const agricultureTitles = titles.filter(title => title.toLowerCase().endsWith('agriculture'));
  const trimmedTitles = agricultureTitles.map(title => title.split(" ")[0]);
  const formattedAgricultureTitles = `${trimmedTitles.join(' and ')} agriculture`;
  const formattedNonAgricultureTitles = `${nonAgricultureTitles.join(', ')}`;
  
  if(!formattedNonAgricultureTitles) return formattedAgricultureTitles;
  return `${[formattedNonAgricultureTitles, formattedAgricultureTitles].join(', ')}`;
```

What I did here is the separation with ` AND ` only the agriculture pressures, and anything else with a comma. The main confusion is that with separating everything with ` AND  ` could give us a false impression that each pressure belongs to the agriculture group:
**BEFORE:**
![image](https://user-images.githubusercontent.com/15097138/76784357-65033900-67ab-11ea-81e0-18dee2dadafa.png)

**AFTER:**
![image](https://user-images.githubusercontent.com/15097138/76784306-4c931e80-67ab-11ea-8443-26317bbe8194.png)

Makes sense?

Reply:
just curiosity, what happens when urban, rangeland and irrigated agriculture are on?

Reply:
All would be separated with a comma:
![image](https://user-images.githubusercontent.com/15097138/76784873-579a7e80-67ac-11ea-81ac-5c642fd7bf52.png)

Reply:
Is it too much to always have an "and" before the last element?

Reply:
I don't think so, but imagine this case:
we've selected three layers:
`RANGELAND, RAINFED AND IRRIGATED AGRICULTURE`
which is fine because layers names are "Rainfed agriculture", "Irrigated agriculture" and "Rangeland"
but if we unclick the "Rainfed agriculture" layer, we will get:
`RANGELAND AND IRRIGATED AGRICULTURE`
what seems to be confusing because we don't have ""Rangeland agriculture" layer, only "Rangeland"
Am I overthinking this? :D

Reply:
I think it is not a problem to have `rangeland and irrigated agriculture`